src/lsp-client/LSPClient.cpp

```diff
@@ -86,6 +86,25 @@ LSPClient::Init(const char *argv[])
 	return stat;
 }
 
+status_t
+LSPClient::IsProcessAlive()
```

Comment:
I think it should return a Boolean instead, in response to the implicit question. Nevertheless, I would suggest moving these two lines in the calling function, unless IsProceasAlive is planned to be used somewhere else.